src/liberty/devCommands.ts

```diff
@@ -163,7 +162,7 @@ export async function removeProject(): Promise<void> {
                     }
                 });
         });
-        
+
```

Comment:
Please do not do any formatting changes to code that is not added or modified by you for this PR.